setup.py

```diff
@@ -59,10 +86,21 @@ def get_basesources(name, osname):
 # Treating some special cases like Octave seems to be the right thing to do,
 # PSYCH_LANGUAGE setting is self-explanatory:
 base_macros = [('PTBOCTAVE3MEX', None), ('PSYCH_LANGUAGE', 'PSYCH_PYTHON')]
-# Disabled: This would enable Py_LIMITED_API, to allow to build one set of modules for all versions of
-# Python >= 3.2. The downside is loss of important functionality [PsychRuntimeEvaluateString()) does not work!].
-# Also, we have build failure on at least Ubuntu 18.04 LTS with Python 3.6, so it is a no-go on Linux for now!
-#base_macros = [('PTBOCTAVE3MEX', None), ('PSYCH_LANGUAGE', 'PSYCH_PYTHON'), ('Py_LIMITED_API', None)]
+
+# if desired by wheel-builder and building on py >= 3.7, build as a limited API wheel.
+if sys.version_info >= (3, 7) and os.environ.get('PTB_LIMITED_WHEEL') == '1':
+    base_macros.append(('Py_LIMITED_API', '0x03070000'))
+    py_limited_api = True
+    # https://github.com/joerick/python-abi3-package-sample/blob/7f05b22b9e0cfb4e60293bc85252e95278a80720/setup.py#L5
```

Comment:
This snippet needs proper license annotation, as it is an almost identical copy of joerick's Apache-2.0 licensed sample code.